bigframes/session/executor.py

```diff
@@ -147,6 +157,35 @@ def export(
             job_config=job_config,
         )
 
+    def export_gcs(
+        self,
+        array_value: bigframes.core.ArrayValue,
+        col_id_overrides: Mapping[str, str],
+        uri: str,
+        format: Literal["JSON", "CSV", "PARQUET"],
+        export_options: Dict[str, Union[bool, str]],
```

Comment:
Any chance to make these consistent? Mapping and Dict. Prefer Mapping as a more general type.